__tests__/app.test.js

```diff
@@ -126,11 +126,18 @@ describe("GET /api/articles", () => {
       comment_count: "2",
     };
 
-    expect(Array.isArray(article)).toBe(true);
-    expect(Array.isArray(article[0])).toBe(false);
-    expect(article[0]).toEqual(expected);
-    expect(article).toHaveLength(13);
-    expect(article).toBeSorted({ key: "created_at", descending: true });
+    const correctStructure = articles.every((article) => {
```

Comment:
This is checking the keys but it would also be beneficial to check the values here too

Reply:
Thanks Beth! I've added an assertion to check each property value of each article is "typeof" string or number etc.